evalml/pipelines/utils.py

```diff
@@ -210,7 +205,8 @@ def generate_pipeline_code(element):
         pipeline_list += ["{} = '{}'".format(k, v)] if isinstance(v, str) else ["{} = {}".format(k, v)]
 
     pipeline_string = "\t" + "\n\t".join(pipeline_list) + "\n" if len(pipeline_list) else ""
-
+    pipeline_string += "\n\tdef __init__(self, parameters, random_seed=0):"
```

Comment:
Need to add init because right now we generate `pipeline(parameters)`, but the first parameter is now `component_graph`. I can file something to track this but I think a discussion that'll be useful to have is, is this still useful as is?

Reply:
If we merge this in, then I think the code generation should return something like `pipeline = BinaryClassifcationPipeline(component_graph=...., )` as opposed to creating subclasses in the style of the old api!